python/az/aro/azext_aro/_params.py

```diff
@@ -122,6 +123,11 @@ def load_arguments(self, _):
                    help='Refresh cluster application credentials.',
                    options_list=['--refresh-credentials'],
                    validator=validate_refresh_cluster_credentials)
+        c.argument('load_balancer_managed_outbound_ip_count',
+                   type=int,
+                   help='The desired number of IPv4 outbound IPs created and managed by Azure for the cluster public load balancer.',  # pylint: disable=line-too-long
+                   validator=validate_load_balancer_managed_outbound_ip_count,
+                   options_list=['--load-balancer-managed-outbound-ip-count', '--lb-ip-count'])
```

Comment:
I saw the shorthand alternative rule as a linter failure from the `azdev` linter commands. It's possible that this rule is ignorable, or that AKS implemented this flag before that rule was imposed and the rule only applies to new and/or changed flags. In any case I'm not sure how we could validate the rule being required until we attempt to upstream this change. 

+1 on aligning the description with AKS, I will make that change.

Reply:
This looks resolved as well, can we resolve?

Reply:
The description is not fixed, but we will be moving that effort to a post-release activity as it will require us to change how the entire `az aro` command is documented (moving user-facing documentation from the parameter definitions themselves into `_help.py`).